### DISCUSSION TITLE
Unable to override archive-product.php

### DISCUSSION THREAD

**Discussion by michal727** [bootscore/bootscore#482] [Q&A]
### Describe the feature that you'd like

Hello.  I wanted to use your template in my woocommerce store but i found some issue.     
According to [https://woocommerce.com/document/template-structure/](https://woocommerce.com/document/template-structure/) 

`If your theme has a woocommerce.php file, you will be unable to override the woocommerce/archive-product.php`

It is unable to remove sidebar from single product template ?  If I remove the sidebar, it will also be removed from the main Shop page. It seems that both single product and main shop page use the same template.   Any fixes? 

### Checklist

- [X] I have searched the issues and this feature has not been requested.

**Comment by crftwrk** [comment]
Yes, use a widget context plugin to hide sidebar in single product pages. See  https://github.com/bootscore/bootscore/issues/150 for instructions how to set sidebar to archive pages only.

**Reply by faridulhassan** [reply]
@crftwrk , 
how can I override the archive-product.php to make custom UI from bootscore child theme ? 
I also need to override the single product page.

Please help.

Thanks.

**Reply by crftwrk** [reply]
> how can I override the archive-product.php to make custom UI from bootscore child theme ?

Not possible, because theme uses a `woocommerce.php` which does not allow to override `archive-product.php`

> I also need to override the single product page.

You can simply copy the `single-product.php` to your child and edit as you need.

See https://developer.woocommerce.com/docs/template-structure-overriding-templates-via-a-theme/

**Reply by crftwrk** [reply]
Check https://github.com/bootscore/bootscore/pull/1073

**Reply by faridulhassan** [reply]
Thanks.
